pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -71,23 +71,51 @@ func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Compon
 		port = int(constants.InferenceServiceDefaultAgentPort)
 	}
 
+	ports := []corev1.ServicePort{
+		{
+			Name: componentMeta.Name,
+			Port: constants.CommonDefaultHttpPort,
+			TargetPort: intstr.IntOrString{
+				Type:   intstr.Int,
+				IntVal: int32(port),
+			},
+			Protocol: corev1.ProtocolTCP,
+		},
+	}
+	for i := 1; i < len(podSpec.Containers[0].Ports); i++ {
```

Comment:
Seems like we are skipping the first port, what happens when gRPC port is specified first so we will potentially have duplicated http ports